pontoon/base/static/js/fluent_interface.js

```diff
@@ -398,7 +398,6 @@ var Pontoon = (function (my) {
         if (
           ast &&
           !ast.attributes.length &&
-          !ast.tags.length &&
```

Comment:
Nit: `ast.{attributes,tags}.length == 0` would make this more explicit.

Reply:
Good point. But since we use such syntax across the entire app a lot, I'd rather keep it.